src/luarocks/fs/lua.lua

```diff
@@ -728,6 +728,14 @@ end
 
 if md5_ok then
 
+-- Support the interface of lmd5 by lhf in addition to md5 by Roberto
+-- and the keplerproject.
+if not md5.sumhexa and md5.digest then
+   md5.sumhexa = function(msg)
+      return md5.digest(msg)
+   end
+end
```

Comment:
`md5.digest` has additional optional arguments. I could have written

```lua
md5.sumhexa = md5.sumhexa or function(msg)
   return md5.digest(msg)
end
```

but I started out with

```lua
if md5.sumhexa then
   print("md5 by roberto")
elseif md5.digest then
   print("lmd5 by lhf")
   md5.sumhexa = function(msg)
      return md5.digest(msg)
   end
else
   error("unsupported md5 module")
end
```

and after removing the debugging output that's what was left.